src/app/actions.js

```diff
@@ -3,7 +3,8 @@ export const actionsList = {
   storeBakery: "STORE_BAKERY",
   storeVisitURL: "STORE_VISIT_URL",
   updateControllerConnection: "UPDATE_CONTROLLER_CONNECTION",
-  logOut: "LOG_OUT"
+  logOut: "LOG_OUT",
+  collapsibleSidebar: "TOGGLE_COLLAPSIBLE_SIDEBAR"
```

Comment:
To support multiple states you could change this to be `SET_SIDEBAR_VISIBILITY` or the like...

The variable names throughout would probably do for an update in that case then.